website/pages/docs/plugins/sources/github/_authentication.md

```diff
@@ -14,3 +14,12 @@ Follow this [guide](https://docs.github.com/en/enterprise-server@3.4/authenticat
 For App authentication, you need to create a GitHub App and install it on your organization. Follow [this guide](https://docs.github.com/en/apps/creating-github-apps/creating-github-apps/creating-a-github-app) and install the App into your organization(s). Give it all the permissions you need (read-only is recommended).
 
 Every organization will have a unique installation ID. You can find it by going to the organization's settings page, and clicking on the "Installed GitHub Apps" tab. The installation ID is the number in the URL of the page.
+
+When providing the `private_key` as a string for the app auth, you will need to replace all the new lines in your PEM file with `\n`. 
```

Comment:
Actually can you try using `|`:

```yaml
- org: cloudquery
  private_key: |
    -----BEGIN RSA PRIVATE KEY-----
    MIIEpQIBAAKCAQEA3eVv6PCn9P8zO+EP8K7pLMfxcA2uVrSZ2f+H3GgYIavDxWtO
    ...
  app_id: your_app_id
```

That might work without replacing the new lines

Reply:
Hi @erezrokah! I tested this out and it works when you are hardcoding the private key. But the newline and indents kind of mess things up when you store the private key in secret manager and try to reference it as a variable.

Reply:
Ah got it. So this is probably related to https://github.com/cloudquery/cloudquery/issues/14599 and partially addressed by https://www.cloudquery.io/docs/advanced-topics/environment-variable-substitution#environment-variables-with-multi-line-json (only for JSON files).

So maybe we should update this PR to say you need to replace newlines with `\n` only if you reference the private key via env variable?

Reply:
Make sense! I've updated it and added more details!